docs/frontend/javascript-approaches/packs.md

```diff
@@ -68,6 +74,28 @@ More reading:
 - [Enhance Jest configuration with Module Aliases](https://alexjover.com/blog/enhance-jest-configuration-with-module-aliases/)
 - [import-js/eslint-plugin-import](https://github.com/import-js/eslint-plugin-import/tree/main/resolvers/webpack)
 
+
+## Initializers in Webpacker
```

Comment:
I think that it may be useful to write a sentence on what an initializer is and why you'd create one.